ligo/src/types.mligo

```diff
@@ -144,6 +146,8 @@ type custom_ep_param = (string * bytes)
 type propose_params =
   { frozen_token : nat
   ; proposal_metadata : proposal_metadata
+  ; name : string
+  ; description : string
```

Comment:
This doesn't look correct.

With the addition of this two values here (and in `proposal`s) every DAO is now forced to handle these two fields.

However, the issue was about adding this information to `RegistryDAO` only, which means that it should have been done with a modification of the `proposal_metadata` handling for that DAO only.

Reply:
I'll look into it. I assume the `voters` field should be handled only in `RegistryDAO` and must be changed as well?

Reply:
No, `voters` should be changed for every kind of DAO, so it's correct for the change to be in `baseDAO`.

You only need to move `name` and `description` to the `proposal_metadata` of RegistryDAO.